social/event/v0.1.0/schema.json

```diff
@@ -14,14 +14,14 @@
         "name": "social_event",
         "indexes": {
             "name": ["name"],
-            "startTime": ["start.date"],
-            "endTime": ["end.date"],
+            "startTime": ["start.dateTime"],
+            "endTime": ["end.dateTime"],
             "source": ["sourceApplication", "sourceId"]
         }
     },
     "layouts": {
-        "create": ["name", "calendarId", "start.date", "end.date", "location"],
-        "view": ["name", "description", "calendarId", "start.date", "end.date", "location", "organizer", "attendees", "status"]
+        "create": ["name", "calendarId", "location"],
+        "view": ["name", "description", "calendarId", "start", "end", "location", "organizer", "attendees", "status"]
     },
     "allOf": [
```

Comment:
Need to add `"type": "object"` as per recommendation in https://chatgpt.com/share/4f8ada15-bcaf-40c6-8365-74d3763fcd7f

Reply:
> Need to add `"type": "object"` as per recommendation in https://chatgpt.com/share/4f8ada15-bcaf-40c6-8365-74d3763fcd7f

Added